src/tasks/classification.py

```diff
@@ -19,14 +18,19 @@ def __init__(self, hparams: DictConfig):
             hparams: Hyperparameters that set in yaml file.
         """
         super().__init__(hparams)
-
-        self.backbone = BACKBONES.get(self._hparams.backbone_name)(**self._hparams.backbone_params)
-
-        self._hparams.pooling_params['in_features'] = self.backbone.get_forward_output_channels()
-        self.pooling = POOLINGS.get(self._hparams.pooling_name)(**self._hparams.pooling_params)
-
-        self._hparams.head_params['in_features'] = self.pooling.get_forward_output_channels()
-        self.head = HEADS.get(self._hparams.head_name)(**self._hparams.head_params)
+        backbones_params = self._hparams.task.params.backbone_params
+        self.backbone = BACKBONES.get(self._hparams.task.params.backbone_name)(**backbones_params)
+
+        pooling_params = self._hparams.task.params.get('pooling_params', dict())
+        pooling_in_features = self.backbone.get_forward_output_channels()
+        pooling_name = self._hparams.task.params.get('pooling_name', 'IdentetyPooling')
+        self.pooling = POOLINGS.get(pooling_name)(in_features=pooling_in_features, **pooling_params)
+        
+        head_params = self._hparams.task.params.get('head_params', dict())
+        head_in_features = self.pooling.get_forward_output_channels()
+        # TODO write IdentetyHead
+        head_name = self._hparams.task.params.get('head_name', 'IdentetyHead')
```

Comment:
Because it's a self._hparams.task.params: Dict, it's not hard structure. May be we need change task config, and write config for backbone pooling and head.

Reply:
Ok, but why do you need TODO here?  
Also, `IdentityHead` is a correct name